experimental/dds/tree2/src/feature-libraries/editable-tree/ProxyTarget.ts

```diff
@@ -75,8 +66,6 @@ export abstract class ProxyTarget<T extends Anchor | FieldAnchor> {
 		return this.lazyCursor;
 	}
 
-	protected abstract buildAnchor(): T;
```

Comment:
there should be implementations of this in both subclasses that can be deleted now.

Reply:
removed `buildAnchor` implementations in subclasses